eng/G11_jumla.md

```diff
@@ -588,11 +588,7 @@ The the remaining syntactic structure is `verb₁–verb₂–object`.
 **mi plis tu kom haus.**
 – I ask you to come home.  
 **plis kom haus!**
-– Please come home!  
-**mi amir tu kluze da mun.**
-– I command you to shut the door.  
-**amir kluze da mun.**
-– Shut the door!
```

Comment:
Thanks for spotting this out. I have never liked the sound of _amir_ in commands because it doesn't sound strong. However, it is good as a content word. Now, when commands are dealt with the auxiliary verb _shal_, this piece of text is not needed.